packages/backend/src/api/request.ts

```diff
@@ -56,6 +56,14 @@ type BuildRequestOptions = {
 };
 export function buildRequest(options: BuildRequestOptions) {
   const requestFn = async <T>(requestOptions: ClerkBackendApiRequestOptions): Promise<ClerkBackendApiResponse<T>> => {
+    // TODO: Remove this once we have a better way to handle ephemeral keys
```

Comment:
I think we may be able to remove this if we use the `clerkMiddlewareRequestDataStore` wrapper - need to test once main is merged in.